src/pages/login/Login.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
+import './Login.scss';
+
+const Login = () => {
+  const [inputValue, setInputValue] = useState({
+    email: '',
+    password: '',
+  });
+
+  const { email, password } = inputValue;
+
+  const handleInput = e => {
+    const { name, value } = e.target;
+    setInputValue({ ...inputValue, [name]: value });
+  };
+
+  const passwordCondition =
+    /^(?=.*[a-zA-Z])(?=.*[!@#$%^*+=-])(?=.*[0-9]).{8,16}$/;
+
+  // const valid = passwordCondition.test(inputValue.password);
+
+  const isValidLogin =
+    email.includes('@') &&
+    email.includes('.') &&
+    passwordCondition.test(inputValue.password);
+
+  const goToSignUp = () => {
+    navigate('/signup');
+  };
+  const navigate = useNavigate();
+
+  const goToMain = e => {
+    e.preventDefault();
+    fetch('http://10.58.2.1:8000/users/signin', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: inputValue.email,
+        password: inputValue.password,
+      }),
```

Comment:
```suggestion
      method: 'POST',
      body: JSON.stringify(inputValue),
```
body에 담기는 값과 state로 관리하는 값이 동일하기 때문에
위와 같이 JSON.stringify에 state 자체를 넣어줘도 동일하게 작동합니다

Reply:
저는 왜 이런 생각을 못하는지 모르겠어요.. 이 부분, 저희 팀원들과도 함께 공유하겠습니다! 감사합니다.